mobile-app/app/components/BottomSheetTokenList.tsx

```diff
@@ -30,79 +33,31 @@ export interface BottomSheetToken {
 export const BottomSheetTokenList = ({
   headerLabel,
   onCloseButtonPress,
-  onTokenPress,
-  navigateToScreen
+  navigateToScreen,
+  collateralTokens
 }: BottomSheetTokenListProps): React.MemoExoticComponent<() => JSX.Element> => memo(() => {
   const { isLight } = useThemeContext()
   const navigation = useNavigation<NavigationProp<BottomSheetWithNavRouteParam>>()
-  const tokenList: BottomSheetToken[] = [
-    {
-      id: 'DFI',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dBTC',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dETH',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dLTC',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dBCH',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dBCH1',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dBCH2',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    },
-    {
-      id: 'dBCH3',
-      name: 'DFI',
-      available: new BigNumber('123'),
-      collateralFactor: new BigNumber(100)
-    }
-  ]
 
   return (
     <BottomSheetFlatList
-      data={tokenList}
-      renderItem={({ item }): JSX.Element => (
+      data={collateralTokens}
+      renderItem={({ item }: { item: CollateralItem }): JSX.Element => (
         <ThemedTouchableOpacity
+          disabled={new BigNumber(item.available).lte(0)}
           onPress={() => {
+            /*
```

Comment:
I added this condition because composite swap has a use case that `onPress` it will not navigate but set the state of the parent

Reply:
Ok noted. Will update this on a separate PR.